### PR TITLE
Setup Exports Page

### PR DESCRIPTION
**Pull request by subinasr**
- Addresses #1963

## Changes

* Setup Exports Page

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: src/newViews/Tagging/Export/index.tsx [hunk 1]
```diff
@@ -1,13 +1,79 @@
...
+    TabList,
+    TabPanel,
+    Container,
+    Button,
+} from '@the-deep/deep-ui';
 
 import Navbar from '../Navbar';
+import styles from './styles.scss';
+
+type ExportType = 'export-history' | 'export-assessment-history';
```
Comment (AdityaKhatri) [inline src/newViews/Tagging/Export/index.tsx]:
```suggestion
type ExportType = 'export-entry-history' | 'export-assessment-history';
```

File: src/newViews/Tagging/Export/index.tsx [hunk 2]
```diff
@@ -1,13 +1,79 @@
...
 
 function Export() {
+    const [activeTab, setActiveTab] = useState<ExportType>('export-history');
```
Comment (AdityaKhatri) [inline src/newViews/Tagging/Export/index.tsx]:
```suggestion
    const [activeTab, setActiveTab] = useState<ExportType>('export-entry-history');
```

File: src/newViews/Tagging/Export/styles.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                flex-grow: 1;
+            }
+
+            .action-buttons {
+                display: flex;
+                align-self: flex-end;
+                flex-shrink: 0;
+
+                .button {
+                    margin: var(--spacing-small);
```
Comment (AdityaKhatri) [inline src/newViews/Tagging/Export/styles.scss]:
```suggestion
                    margin: var(--dui-spacing-small);
```

**Review by AdityaKhatri — DISMISSED** [review]

File: src/newViews/Tagging/Export/index.tsx [hunk 4]
```diff
@@ -1,13 +1,79 @@
...
+                            </Tab>
+                        </TabList>
+
+                        <div className={styles.actionButtons}>
+                            <Button
+                                name="export-entry"
+                                className={styles.button}
+                                onClick={handleExportEntryClick}
+                                icons={<IoAdd />}
+                                variant="secondary"
```
Comment (tnagorra) [inline src/newViews/Tagging/Export/index.tsx:52]: Please disable this button

File: src/newViews/Tagging/Export/index.tsx [hunk 5]
```diff
@@ -1,13 +1,79 @@
...
+                            >
+                                New Export
+                            </Button>
+                            <Button
+                                name="export-assessment"
+                                className={styles.button}
+                                onClick={handleExportAssessmentClick}
+                                icons={<IoAdd />}
+                                variant="tertiary"
```
Comment (tnagorra) [inline src/newViews/Tagging/Export/index.tsx:62]: Please disable this button

**Review by tnagorra — CHANGES_REQUESTED** [review]

**Review by tnagorra — DISMISSED** [review]

**Review by samshara — DISMISSED** [review]

**Review by samshara — APPROVED** [review]

**Pull request merged**